app/src/main/java/org/mozilla/focus/telemetry/TelemetryIntegration.kt

```diff
@@ -41,8 +33,13 @@ private const val KEY_CLICKED_HOME_TILE_IDS_PER_SESSION = "clickedHomeTileIDsPer
         "TooManyFunctions",
         "LargeClass"
 )
-object TelemetryWrapper {
-    private const val TELEMETRY_APP_NAME_FOCUS_TV = "FirefoxForFireTV"
+open class TelemetryIntegration protected constructor(
+    private val sentryIntegration: SentryIntegration = SentryIntegration
+) {
+
+    companion object {
+        val INSTANCE: TelemetryIntegration by lazy { TelemetryIntegration() }
```

Comment:
I don't love that we have to do this but it looks like the best of all of our options:
- (old solution) Top-level object with static `init()`: can't replace the instance under testing; strange mechanics when testing the object
- Give TelemetryIntegration instance to Application to own: need context to add telemetry